src/main/java/in/projecteka/datanotificationsubscription/subscription/SubscriptionRequestRepository.java

```diff
@@ -281,4 +288,8 @@ private HipDetail buildHip(String hipId) {
                             });
                 }));
     }
+
+    private String joinByComma(Collection<String> list) {
+        return list.stream().map(e -> String.format("'%s'", e)).collect(Collectors.joining(", "));
```

Comment:
```suggestion
        return String.join(",", list);
```
Also please rename the parameter.